Self_Check.py

```diff
@@ -0,0 +1,79 @@
+'''
+This code takes an OpenAddresses file with zip codes
```

Comment:
Is this true? Technically, I think, it's not an OpenAddresses file. Instead it's a file created by `Pull_only_zip.py`.

Yes, nearly the same thing except that running `Pull_only_zip.py` *should* guarantee that the rows contain *good* data. If you were to use an OpenAddresses file with this pre-processing this process would/could fail.